### ISSUE TITLE
premature column wrapping

### ISSUE THREAD

**Issue by brho** [u-root/cpu#63]
when in a large terminal, if i run bash or sh with cpu, my terminal will be "prematurely wrapped".  meaning that instead of using the full window size of my host, it'll wrap to 70 characters.

e.g.
```
$ ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
$ ./cpu TARGET bash --norc --noprofile
bash-5.1# fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
ffffff
bash-5.1# resize
COLUMNS=164;
LINES=46;
export COLUMNS LINES;
bash-5.1# stty -a
speed 38400 baud; rows 46; columns 164; line = 0;
```

both on the target machine (natively), on my host, and in cpu, bash thinks there are 164 columns (which is correct).  stty -a agrees.

**Comment by rminnich** [comment]
I think this needs a specific ssh message which I'll implement.

**Comment by rminnich** [comment]
fixed by #67
